src/Microsoft.Diagnostics.Runtime/src/Linux/LinuxLiveDataReader.cs

```diff
@@ -84,7 +84,7 @@ private void Dispose(bool _)
 
             if (_suspended)
             {
-                int status = (int)ptrace(PTRACE_DETACH, (int)ProcessId, IntPtr.Zero, IntPtr.Zero);
+                int status = (int)ptrace(PTRACE_DETACH, ProcessId, IntPtr.Zero, IntPtr.Zero);
```

Comment:
Unrelated to this change, maybe we should simply define `ptrace` and `process_vm_readv` as returning int instead of casting?

Reply:
Yeah we could do that.